### PR TITLE
add the asynchronous linear filter paper tpami23, add HDR event-frame dataset, SHEF stereo dataset, flicker removal dataset

### PR DESCRIPTION
**Pull request by ziweiWWANG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -973,8 +970,12 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [Code](https://
 - <a name="Freeman23mmsys"></a>Freeman, A., Singh, M., Mayer-Patel, K.,  
 *[An Asynchronous Intensity Representation for Framed and Event Video Sources](https://doi.org/10.1145/3587819.3590969)*,  
 ACM Multimedia Systems (MMSys), 2023. [PDF](https://arxiv.org/abs/2301.08783), [Code](https://github.com/ac-freeman/adder-codec-rs/).
-
-
+- <a name="Wang23tpami"></a>Wang, Z., Ng, Y., Scheerlinck, C., Mahony., R.,  
+*[An Asynchronous Linear Filter Architecture for Hybrid Event-Frame Cameras](https://openaccess.thecvf.com/content/ICCV2021/html/Wang_An_Asynchronous_Kalman_Filter_for_Hybrid_Event_Cameras_ICCV_2021_paper.html)*,  
+IEEE Transactions on Pattern Analysis and Machine Intelligence (TPAMI), 2023. [PDF](https://arxiv.org/pdf/2309.01159.pdf), [Project Page](https://github.com/ziweiWWANG/Event-Asynchronous-Filter), [YouTube](https://www.youtube.com/watch?v=K1kRjw9xQNU).
```
Comment (guillermogb) [inline README.md:975]: IEEE Trans. Pattern Anal. Machine Intell. (TPAMI)

File: README.md [hunk 2]
```diff
@@ -973,8 +970,12 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [Code](https://
 - <a name="Freeman23mmsys"></a>Freeman, A., Singh, M., Mayer-Patel, K.,  
 *[An Asynchronous Intensity Representation for Framed and Event Video Sources](https://doi.org/10.1145/3587819.3590969)*,  
 ACM Multimedia Systems (MMSys), 2023. [PDF](https://arxiv.org/abs/2301.08783), [Code](https://github.com/ac-freeman/adder-codec-rs/).
-
-
+- <a name="Wang23tpami"></a>Wang, Z., Ng, Y., Scheerlinck, C., Mahony., R.,  
+*[An Asynchronous Linear Filter Architecture for Hybrid Event-Frame Cameras](https://openaccess.thecvf.com/content/ICCV2021/html/Wang_An_Asynchronous_Kalman_Filter_for_Hybrid_Event_Cameras_ICCV_2021_paper.html)*,
```
Comment (guillermogb) [inline README.md:974]: I think this link should be the official one from TPAMI:
https://doi.org/10.1109/TPAMI.2023.3311534

File: README.md (hunk 1 above)
Comment (guillermogb) [reply README.md:975]: BTW, Ziwei, were you aware of this TPAMI paper https://doi.org/10.1109/TPAMI.2022.3230727 ?

**Review by guillermogb — CHANGES_REQUESTED** [review]
congratulations on the TPAMI paper!

File: README.md [hunk 3]
```diff
@@ -971,8 +971,8 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [Code](https://
 *[An Asynchronous Intensity Representation for Framed and Event Video Sources](https://doi.org/10.1145/3587819.3590969)*,  
 ACM Multimedia Systems (MMSys), 2023. [PDF](https://arxiv.org/abs/2301.08783), [Code](https://github.com/ac-freeman/adder-codec-rs/).
 - <a name="Wang23tpami"></a>Wang, Z., Ng, Y., Scheerlinck, C., Mahony., R.,  
-*[An Asynchronous Linear Filter Architecture for Hybrid Event-Frame Cameras](https://openaccess.thecvf.com/content/ICCV2021/html/Wang_An_Asynchronous_Kalman_Filter_for_Hybrid_Event_Cameras_ICCV_2021_paper.html)*,  
-IEEE Transactions on Pattern Analysis and Machine Intelligence (TPAMI), 2023. [PDF](https://arxiv.org/pdf/2309.01159.pdf), [Project Page](https://github.com/ziweiWWANG/Event-Asynchronous-Filter), [YouTube](https://www.youtube.com/watch?v=K1kRjw9xQNU).
+*[An Asynchronous Linear Filter Architecture for Hybrid Event-Frame Cameras](https://ieeexplore.ieee.org/document/10238826)*,
```
Comment (guillermogb) [inline README.md:974]: it's still better to use the DOI link ;)

**Review by guillermogb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ziweiWWANG) [reply README.md]: Hi Guillermo, I appreciate this! I wasn't aware of this paper. Excellent work! I'll take a look at it right away :)

**Pull request merged**